tests/theme_settings.spec.ts

```diff
@@ -365,4 +368,62 @@ describe('Module: theme settings validation (config/settings_schema.json)', () =
       ]);
     });
   });
+
+  describe('Unit: styles', () => {
```

Comment:
Nit: Could we get a happy-path test case and also have versions of these tests for the other panel types?

Reply:
good idea, added a bunch more tests